packages/mobile/src/backup/BackupQuiz.tsx

```diff
@@ -44,19 +54,23 @@ interface DispatchProps {
 type Props = WithTranslation & DispatchProps & NavigationInjectedProps
 
 export class BackupQuiz extends React.Component<Props, State> {
-  static navigationOptions = () => ({
-    ...headerWithBackButton,
+  static navigationOptions = ({ navigation }: NavigationInjectedProps) => ({
+    ...headerWithCancelButton, // TODO switch to v2 once Ivans PR is in
+    headerRight: <DeleteWord onPressBackspace={navigation.getParam('onPressBackspace')} />,
```

Comment:
we could add a check here that `navigation.getParam('onPressBackspace')` is not null, at least until TS can do it for us instead.

Reply:
done